x-pack/plugins/security_solution/public/timelines/components/edit_data_provider/index.tsx

```diff
@@ -193,18 +192,16 @@ export const StatefulEditDataProvider = React.memo<Props>(
             <EuiFlexGroup direction="row" gutterSize="none" justifyContent="spaceBetween">
               <EuiFlexItem grow={false}>
                 <EuiFormRow label={i18n.FIELD}>
-                  <EuiToolTip content={updatedField.length > 0 ? updatedField[0].label : null}>
-                    <EuiComboBox
-                      data-test-subj="field"
-                      isClearable={false}
-                      onChange={onFieldSelected}
-                      options={getCategorizedFieldNames(browserFields)}
-                      placeholder={i18n.FIELD_PLACEHOLDER}
-                      selectedOptions={updatedField}
-                      singleSelection={{ asPlainText: true }}
-                      style={{ width: `${FIELD_COMBO_BOX_WIDTH}px` }}
-                    />
-                  </EuiToolTip>
+                  <EuiComboBox
+                    data-test-subj="field"
+                    isClearable={false}
+                    onChange={onFieldSelected}
+                    options={getCategorizedFieldNames(browserFields)}
+                    placeholder={i18n.FIELD_PLACEHOLDER}
+                    selectedOptions={updatedField}
+                    singleSelection={{ asPlainText: true }}
+                    style={{ width: `${FIELD_COMBO_BOX_WIDTH}px` }}
```

Comment:
:grey_question: Should this be handled with a styled component?

Reply:
yes, but now I'm focused to fix only the bug that is related to the `7.10` release